projects/Exceptions/ElementsList/src/com/valentin/EmpyList.java

```diff
@@ -0,0 +1,7 @@
+package com.valentin;
+
+public class EmpyList extends IndexOutOfBoundsException {
```

Comment:
ListIsEmptyException sounds better. Exceptional classes in most cases end with Exception